### ISSUE TITLE
Update to latest espresso?

### ISSUE THREAD

**Issue by tmtrademarked** [airbnb/okreplay#103]
Hey there! We're trying to update our app to use Espresso 3.2, and it seems like OkReplay currently blocks this upgrade because the dependency on Espresso is strict. Possibly this is due to okreplay-espresso using an API dependency on espresso:3.1.0.

Is there any chance this library can be updated to use a more modern espresso, or at least to not prohibit such? I'm not sure how much work that would involve, but it seems like it might make it easier for applications to continue using it.

**Comment by felipecsl** [comment]
absolutely, i don't see any reasons why we wouldn't be able to bump the espresso depedency

**Comment by adk** [comment]
Hello @rossbacher, are there any updates on this issue? Is this project still being maintained?

**Comment by radityagumay** [comment]
@felipecsl hi any updates on this?
